themes/base.theme.bash

```diff
@@ -191,6 +194,12 @@ function git_prompt_minimal_info {
 }
 
 function git_prompt_vars {
+  if ${SCM_GIT_USE_GITSTATUSD} && _command_exists gitstatus_query && gitstatus_query && [[ "${VCS_STATUS_RESULT}" == "ok-sync" ]]; then # use faster gitstatusd
```

Comment:
`${SCM_GIT_USE_GITSTATUSD}` is true if unset. Is this what you want?

Reply:
It will be set by `base.theme.bash`